### PR TITLE
New examples

### PR DESCRIPTION
**Pull request by lulldev**
#177

### CODE REVIEW

File: examples/type-to-search-wiki/src/app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const getSearchResult = pipe([
+  toSearchUrl,
+  fetchJsonp,
+  response => response.json(),
+])
+
+const selectFoundedNames = result => result[1]
+
+const render = resultContent => {
+  resultList.innerHTML = reduce((html, item) => html + template.replace(/\{name\}/g, item), '')(resultContent)
```
Comment (richytong) [inline examples/type-to-search-wiki/src/app.js:23]: i would avoid using a rubico function when the array property function would read more naturally here

```
result.innerHTML = resultContent.reduce...
```

**Review by richytong — CHANGES_REQUESTED** [review]
If the example is designed to run in the browser let's just use ESM over any bundlers

**Comment by richytong** [comment]
by the way @lulldev you aren't showing up in the contributors list most likely because your email isn't added to your github account. You should add the one associated with this commit https://github.com/a-synchronous/rubico/commit/ff1ec6167947268f9f5a6540c64d068f77bbcde3.patch

**Comment by lulldev** [comment]
@richytong about esm. can be more?

**Review by richytong — CHANGES_REQUESTED** [review]
moving everything to ESM means we should replace all the requires with import statements, as well as all the module.exports statements with export default. We should also rename the `src` folder to `public`. In order to import rubico in the browser I recommend importing the core globally by adding this script to the head of the index.html

```html
<script defer src="https://unpkg.com/rubico@1.6.17/dist/rubico.min.js" crossorigin></script>
```

**Review by richytong — CHANGES_REQUESTED** [review]
also with ESM we don't need a build step anymore. All we have to do is serve the `public` folder with a static file server like [serve](https://www.npmjs.com/package/serve)

**Comment by richytong** [comment]
also about your email I think you should look into making the one in your commits match up to your actual email. This is done through creating a new gpg key and adding it to your github gpg keys. This is a pretty good guide for it https://docs.github.com/en/enterprise-server@3.1/authentication/managing-commit-signature-verification/generating-a-new-gpg-key

**Review by richytong — APPROVED** [review]
all right, if you wanna keep src that's fine. I also ran your example, everything looks good.

LGTM

**Pull request merged**